install/installer/pkg/components/ide-metrics/configmap.go

```diff
@@ -369,6 +369,26 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 				},
 			},
 		},
+		{
```

Comment:
We discussed:
- to remove metrics from this PR
- move log test only to the integration test
- add supervisor_jb_backend_plugin_started_total to the backend plugin itself in another PR that it is more reliable
- drop supervisor_jb_backend_plugin_incompatible_total completely, since we can cover with SLO plugin vs backend startups, and after that look at logs